src/Tizen.NUI/src/public/BaseComponents/ControlState.cs

```diff
@@ -286,7 +286,7 @@ public override string ToString()
 
             var rest = lhs.stateList.Except(rhs.stateList);
 
-            if (rest.Count() == 0)
+            if (rest.Any())
```

Comment:
You forgot `!`.
```suggestion
            if (!rest.Any())
```

BTW, what do you think of using count local variable?
Since `rest.Count()` is used in the line below, we can improve performance by using count local variable.
(IEnumerable<T>.Count() is O(n) Operation.(https://referencesource.microsoft.com/#System.Core/System/Linq/Enumerable.cs,41ef9e39e54d0d0b))

Reply:
Thank you for your suggestion! I like it! and accept your kind guide. :)